Backend/src/main/java/com/abhinternship/auctionapp/model/Bid.java

```diff
@@ -0,0 +1,72 @@
+package com.abhinternship.auctionapp.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+@Table(name = "bid")
+public class Bid {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    @NotBlank
+    private String price;
+
+    @ManyToOne (fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "bidder_id", nullable = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JsonIgnore
```

Comment:
Why is this JsonIngored? You realize this will ignore this property when fetching the object from frontend?